api/views.py

```diff
@@ -6,17 +6,58 @@
 from django.db.models import Q
 from django.http import HttpResponse
 from django.core import serializers
+import logging
 import json
-import pickle
+
 
 from comments.models import Comment
 from friends.models import Friend, Follow
-from posts.models import Post
+from posts.models import Post, PostEncoder
 from tags.models import Tag
 from django.contrib.auth.models import User
 from django.http import JsonResponse
 from django.views.decorators.http import require_http_methods
 
+
+#TODO Move to another file...
+from functools import wraps
+from django.utils.decorators import available_attrs
+logger = logging.getLogger('django.request')
+
+def require_http_accept(request_accept_list):
```

Comment:
What is the best place/ best practice for a spot to put this... Could be useful else where maybe? I might also generalize this for any header.